File: JS_Challenge/3c.js
const vowels = ['a', 'e', 'i', 'o', 'u'];
function vowel_counter(word) {
    let counter = 0;
    for (let i = 0; i < word.length; i++) {
        if (vowels.includes(word.toLowerCase()[i])) {
            counter++;
        }
    }
    return counter;
}
console.log(vowel_counter("Hello World"));

File: Python_Challenge/3c.py
# Write a function in Python that accepts a string and returns the number of vowels in the string.
vowels = ['a', 'e', 'i', 'o', 'u']


def vowel_counter(word):
    counter = 0
    for i in word.lower():
        if i in vowels:
            counter += 1
    return counter

print(vowel_counter('Hello World'))